grades/models_test.py

```diff
@@ -139,3 +142,31 @@ def test_for_user_course(self):
         grades = ProctoredExamGrade.for_user_course(user, course)
 
         assert list(grades) == [available_grade]
+
+    @ddt.data(
+        (5.0, True, EXAM_GRADE_PASS),
+        (0.0, True, EXAM_GRADE_PASS),
+        (-5.0, False, EXAM_GRADE_FAIL),
+    )
+    @ddt.unpack
+    def test_set_score(self, grade_adjust, expected_passed_value, expected_grade_str):
+        """Tests that the set_score helper method sets score-related fields appropriately"""
+        passing_score = 60.0
+        grade = ProctoredExamGradeFactory.build(
+            passing_score=passing_score,
+            score=None,
```

Comment:
`set_score` sets all 3 values, so I figured a good way to assert that `set_score` was doing its job was to initialize the object with all of those values initially set to `None`